plugins/MauticFocusBundle/EventListener/FocusSubscriber.php

```diff
@@ -134,7 +129,7 @@ public function onKernelRequest(GetResponseEvent $event)
             $formGenerateUrl = $this->router->generate('mautic_form_generateform');
 
             if (strpos($requestUri, $formGenerateUrl) !== false) {
-                $id = InputHelper::_($this->request->get('id'));
+                $id = InputHelper::_($this->requestStack->getCurrentRequest()->get('id'));
```

Comment:
So the current request could be null if this is run from a CLI environment. Probably not something we need to deal with here.

Reply:
It's exactly the same as the state before this change. This PR does not introduce a new bug. In worst case, it keeps the old one.